test/index.test.js

```diff
@@ -1,98 +1,193 @@
 require('env2')('.env');
 var sendemail = require('../lib/index.js'); // auto-set TEMPLATE_DIR
-var email     = sendemail.email;
-var path      = require('path');
-var test      = require('tape');
-var dir       = __dirname.split('/')[__dirname.split('/').length-1];
-var file      = dir + __filename.replace(__dirname, '');
-var decache   = require('decache');
+var email = sendemail.email;
+var sendMany = sendemail.sendMany;
+// var set_http_proxy = sendemail.set_http_proxy;
```

Comment:
are we keeping proxy in?
I would _much_ prefer this to be raised a _separate_ issue where the "Corporate Firewall" use-case is specifically addressed. Preferably with reference to an AWS forum thread/topic covering _why_ a proxy is required to use the `aws-sdk` ... we've used the sdk in quite a few companies with _strict_ firewalls without requiring a proxy... but we're all ears if there's a use-case. 👍